packages/headless/.gitignore

```diff
@@ -32,4 +32,11 @@ lerna-debug.log*
 !.vscode/settings.json
 !.vscode/tasks.json
 !.vscode/launch.json
-!.vscode/extensions.json
\ No newline at end of file
+!.vscode/extensions.json
+
+#env
+.env.development
+.env.local
+.env.staging
+.env.uat
+.env.production
```

Comment:
We don't need separate file for env. for now you can remove these

Reply:
ok.done